docs/technical-design/form-data-flow.md

```diff
@@ -0,0 +1,30 @@
+# Form-data flow
+
+The MC-Review site's primary function is to collect form data from states, and package it for review by CMS. In this document, we'll describe the flow of that form data through the app; from creation to storage, and back from storage to the UI.
+
+## Information entry
+
+```mermaid
+graph TD
+    A["User enters form data"] -->|"validated by Formik and Yup"| B["Data submit (by clicking 'continue' or 'save as draft')"]
+    B -->|"convert data to a base64 protocol buffer<br />(toProtoBuffer)"| C["GraphQL resolver"]
+    C -->|"convert data back to domain models for various checks<br />(toDomain)"| D["Prisma (postgres)"]
```

Comment:
```suggestion
    C -->|"convert data back to  backend domain models for various checks<br />(toDomain)"| D["Prisma (postgres)"]
```

Added the work "backend" to domain models since we have both frontend and backend domain models